lib/datasource/git-tags/index.js

```diff
@@ -3,39 +3,50 @@ const { getRemoteInfo } = require('isomorphic-git');
 const cacheNamespace = 'git-tags';
 const cacheMinutes = 10;
 
-async function getPkgReleases({ lookupName }) {
+async function getPkgReleases({ lookupName, registryUrls = [] }) {
```

Comment:
Can you describe how these semantics works? e.g. it's possible that a single dependency (`depName`) may have multiple repositories (`registryUrls`) to look it up in? Is this a requirement from Swift?

Reply:
What if you instead store publisher/repo into `depName` and the full URL into `lookupName`?

Reply:
Ah, makes sense. Just followed other datasources code I knew.